src/modules/launcher/Plugins/Microsoft.Plugin.Folder/ContextMenuLoader.cs

```diff
@@ -59,6 +60,39 @@ public List<ContextMenuResult> LoadContextMenus(Result selectedResult)
                         }
                     }
                 });
+
+                contextMenus.Add(new ContextMenuResult
+                {
+                    PluginName = Assembly.GetExecutingAssembly().GetName().Name,
+                    Title = _context.API.GetTranslation("Microsoft_plugin_folder_open_in_console"),
+                    Glyph = "\xE756",
+                    FontFamily = "Segoe MDL2 Assets",
+                    SubTitle = $"Open current {fileOrFolder} path in console",
+                    AcceleratorKey = Key.C,
+                    AcceleratorModifiers = ModifierKeys.Control | ModifierKeys.Shift,
+
+                    Action = (context) =>
+                    {
+                        try
+                        {
+                            if (fileOrFolder == "file")
```

Comment:
why do we use hardcoded string for checking boolean condition? 
Should be something like -convert that string into bool
string fileOrFolder  => bool isFolder